kgo.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const foreign = require('foreign');
+const kgo = require('kgo');
+const S = require('sanctuary');
+const shuv = require('shuv');
+
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const complete = (error, result) => {
+  if(error == null){
+    process.stdout.write(result);
+    process.exit(0);
+  }else{
+    process.stderr.write(String(error) + '\n');
+    process.exit(1);
+  }
+};
+
+const main = () => {
+  const relative = (filename) => path.join(process.argv[2], filename);
```

Comment:
Since there's exactly one parameter, we can drop the parens (i.e. `filename => …`).